src/org/usfirst/frc/team2706/robot/controls/operatorFeedback/Rumbler.java

```diff
@@ -0,0 +1,127 @@
+package org.usfirst.frc.team2706.robot.controls.operatorFeedback;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class Rumbler extends Command {
+    
+    public static final int DRIVER_JOYSTICK = 1;
+    public static final int OPERATOR_JOYSTICK = 2;
+    public static final int BOTH_JOYSTICKS = 3;
+    
+    /* An integer which is one DRIVER_JOYSTICK, OPERATOR_JOYSTICK or BOTH_JOYSTICKS
+     * values. Represents which controller to rumble
+     */ 
+    int controllerToRumble;
+    
+    // How long to rumble
+    double timeOn;
+    
+    // How much break between rumbles
+    double timeOff;
+    
+    // How many times to repeat the pattern
+    int repeatCount;
+    
+    // The joysticks that will be rumbled
+    Joystick driver;
+    Joystick operator;
+    
+    // The last time in seconds of an event.
+    double timePoint;
+    
+    // States of the pattern
+    private static final int RUMBLE = 0;
+    private static final int BREAK = 1;
+
+    // Indicate what part of the pattern we're in, RUMBLE or BREAK
+    int state;
+    
+    
+    /**
+     * Class to rumble the controllers of the robot with the purpose of 
+     * giving haptic feedback to the drivers for important events.<p>
+     * @param timeOn How long to rumble <p>
+     * @param timeOff How long to pause between rumbles <p>
+     * @param repeatCount How long to repeat the pattern <p>
+     * @param controllerToRumble Which controller (one of
+     * DRIVER_JOYSTICK, OPERATOR_JOYSTICK or BOTH_JOYSTICKS) to rumble.
+     */
+    public Rumbler(double timeOn, double timeOff, int repeatCount, int controllerToRumble) {
+        this.timeOn = timeOn;
+        this.timeOff = timeOff;
+        this.repeatCount = repeatCount;
+        
+        driver = Robot.oi.getDriverJoystick();
+        operator = Robot.oi.getOperatorJoystick();
+        
+        this.controllerToRumble = controllerToRumble;
+        
+        // Begin rumbling
+        rumble(true);
+        
+        // Add this command to the scheduler.
+        start();
```

Comment:
I'm not sure, I thought it just made it easier. In case the person forgets to do `start()` because if they do forget it then you end up with controllers that rumble infinitely because the `end()` function is never called.